hugo/layouts/get-started/single.html

```diff
@@ -0,0 +1,102 @@
+                    <a data-scroll="no" class="nav-link h5 h-100 active" id="startTab01-tab" data-toggle="tab" href="#startTab01" role="tab" aria-controls="startTab01" aria-selected="true">Engineers / Researchers</a>
+                </li>
+                <li class="nav-item col-sm-12 col-md-4">
+                    <a data-scroll="no" class="nav-link h5 h-100" id="startTab02-tab" data-toggle="tab" href="#startTab02" role="tab" aria-controls="startTab02" aria-selected="false">Engineering managers</a>
+                </li>
+                <li class="nav-item col-sm-12 col-md-4">
+                    <a data-scroll="no" class="nav-link h5 h-100" id="startTab03-tab" data-toggle="tab" href="#startTab03" role="tab" aria-controls="startTab03" aria-selected="false">Executives</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</header>
+
+<section id="engineering" class="pt-5 pb-0 pb-m-5">
+    <div class="container text-dark">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="tab-content" id="startTabContent">
+                    <div class="tab-pane fade show active" id="startTab01" role="tabpanel" aria-labelledby="startTab01-tab">
+                        <div class="row py-4">
+                            <div class="col-sm-12 col-lg-8 mb-4 mb-md-0">
+                                <h2 class="title-line line-yellow">Engineers / Researchers</h2>
+                                <p>With source{d} you can start getting answers to tough questions in a matter of minutes with simple SQL queries. source{d} supports different interfaces that adapt to different stages of development, workflows and use cases. Enjoy the flexibility to choose whether to work on our built-in interactive web interface, your favorite SQL client, command-line interface or a combination of these. Whether you're an Open Source project maintainer or tech lead for a project at work you can use source{d} to better understand what's going on in your codebases, its evolution over time and inform decisions on what needs to happen next!</p>
+                                <p>source{d} is part of a growing community of academic researchers interested in the fields of Software Repository mining, programming language analysis or Machine Learning on Code. We develop Open Source platform components, datasets, and Machine Learning models that researchers can use as part of their research projects The source{d} team also curates a list of tools and research papers related to Machine Learning on Code that we also share via our MLonCode newsletter!</p>
+                            </div>
+                            <div class="col-sm-12 col-lg-4 text-center">
+                                <a href="https://go.sourced.tech/engine-download" class="btn btn-highlight w-100 mb-4">Download source{d}</a>
+                                <a href="https://docs.sourced.tech" class="btn btn-highlight w-100 mb-4">Documentation</a>
```

Comment:
idk if we really want to link kind of outdated docs
https://docs.sourced.tech/intro/
wdyt @vcoisne ?